src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx

```diff
@@ -167,18 +180,62 @@ export const ConversationsList = ({
     );
   };
 
+  const getFilteredGroupConversations = () => {
```

Comment:
Would be better if we move this to another component.
I think @bardiawire is also suggesting the same.